src/openforms/translations/tests/test_api.py

```diff
@@ -0,0 +1,131 @@
+from collections import namedtuple
+
+from django.test import override_settings
+from django.urls import reverse
+from django.utils.translation import gettext as _
+
+from hypothesis import given, settings, strategies as st
+
+# from hypothesis.extra.django import TestCase
+from rest_framework.test import APISimpleTestCase
+
+Language = namedtuple("Language", "conf local")
+
+
+class I18NAPITests(APISimpleTestCase):
+    databases = ["default"]
+
+    @given(
+        st.lists(
+            st.sampled_from(
+                [
+                    Language(("en", _("English")), "English"),
+                    Language(("nl", _("Dutch")), "Nederlands"),
+                    Language(("tr", _("Turkish")), "Türkçe"),
+                ]
+            ),
+            unique=True,
+        )
+    )
+    @settings(deadline=500)
+    def test_info_contains_available_langauges(self, languages):
+        with override_settings(
+            USE_I18N=True,
```

Comment:
don't override settings that are already the current value (and cannot be altered by the env):

* changing the setting from `True` to `False` would break the functionality at runtime for real deploys
* having it overridden here would *not* point out this misconfiguration. if someone messes up with the (default) settings, CI needs to fail